### DISCUSSION TITLE
Move Extensions/Overflow Menu to bottom of Sidebar

### DISCUSSION THREAD

**Discussion by oerix** [zen-browser/desktop#3641] [Ideas]
It would be great if you could add the option to add the Settings Overflow menu, and Extensions (including pinned) icons to the bottom of the Sidebar. Currently, it's not possible to move these icon using the 'Customize Toolbar' option.

With the new beta update, the 'Single layout' is enormously wide, and the only way to slim it down is to remove icons. If we could shift them to the bottom we can better organise it and reduce the width.

Thanks

**Comment by SamuXzX** [comment]
The Extensions menu disappeared recently, in favour of the "unified panel", so there's not much way of customising where you access extensions 🤷🏻‍♂️
I talk about this here: https://github.com/zen-browser/desktop/discussions/11014
